tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -153,18 +148,25 @@ def parse_args():
         '''
     )
 
-    parser.add_argument('--use_nnapi', action='store_true',
+    ep_args = parser.add_mutually_exclusive_group()
```

Comment:
these aren't mutually exclusive.

Reply:
This change is dropped.

- [ ] Reimpl it on the new `convert_onnx_models_to_ort.py`